streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -274,31 +317,15 @@ public Sensor addThroughputSensor(final String scopeName,
                                       final Sensor.RecordingLevel recordingLevel,
                                       final String... tags) {
 
-        return addThroughputSensor(null,
-                                   scopeName,
-                                   entityName,
-                                   operationName,
-                                   recordingLevel,
-                                   tags);
-
-    }
-
-    public Sensor addThroughputSensor(final String taskName,
-                                      final String scopeName,
-                                      final String entityName,
-                                      final String operationName,
-                                      final Sensor.RecordingLevel recordingLevel,
-                                      final String... tags) {
-
         final Map<String, String> tagMap = constructTags(scopeName, entityName, tags);
         final Map<String, String> allTagMap = constructTags(scopeName, "all", tags);
 
         // first add the global operation metrics if not yet, with the global tags only
-        final Sensor parent = metrics.sensor(sensorName(buildUniqueSensorName(operationName, taskName), null), recordingLevel);
+        final Sensor parent = metrics.sensor(threadName + "." + operationName, recordingLevel);
```

Comment:
Maybe I am missing something, but does this generate the same name as before? Where does `taskName` come from now?

Reply:
`taskName` was always null, and was therefore never affecting the name of the sensor.